addons/stock/models/stock_warehouse.py

```diff
@@ -323,37 +320,27 @@ def _create_or_update_delivery_route(self, routes_data):
             if warehouse.delivery_route_id:
                 delivery_route = warehouse.delivery_route_id
                 delivery_route.write({'name': warehouse._format_routename(route_type=warehouse.delivery_steps)})
-                delivery_route.pull_ids.write({'active': False})
+                delivery_route.rule_ids.write({'active': False})
```

Comment:
filter on pull
what about pull_push? :)

Reply:
not filter on pull. It will archive all the rules on the route. Then modify the route name and search or create all the default rules for depends the delivery steps